src/Tickets/Emails/Email_Template.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * Handles registering and setup for the Tickets Emails settings tab.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+use Tribe__Template;
+use Tribe__Tickets__Main;
+
+/**
+ * Class Email_Template
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Email_Template {
+
+	private bool $preview = false;
+
+	private Array $context_data = [];
+	
+	/**
+	 * Gets the template instance used to setup the rendering html.
+	 *
+	 * @since TBD
+	 *
+	 * @return Tribe__Template The template object.
+	 */
+	public function get_template(): Tribe__Template {
+		if ( empty( $this->template ) ) {
+			$this->template = new Tribe__Template();
+			$this->template->set_template_origin( Tribe__Tickets__Main::instance() );
+			$this->template->set_template_folder( 'src/admin-views/settings/emails' );
```

Comment:
For now we can use this path within the `admin-views`, but before release we'll have to move the templates to the public `src/views/v2` in order to make it consistent with our other views, given that these templates will have to be overridable. Maybe we can leave a `@todo` for this.